targets/targets.json

```diff
@@ -1497,7 +1499,10 @@
             "MPU"
         ],
         "bootloader_supported": true,
-        "device_name": "STM32F303RE"
+        "device_name": "STM32F303RE",
+        "supported_application_profiles": [
```

Comment:
Remove these lines and add in "MCU_STM32"

Reply:
Hi @evedon, are all the STM32 MCUs able to be used in bare-metal mode? 
I believe we haven't fully tested all of them? is there anything blocking them to be used in baremetal mode?